common/persistence/sql/sqlplugin/visibility.go

```diff
@@ -42,6 +42,7 @@ type (
 		ExecutionDuration    *time.Duration
 		StateTransitionCount *int64
 		Memo                 []byte
+		ChasmMemo            []byte
```

Comment:
We are no longer adding a new column to the SQL DB schema. This is not necessary anymore.

Reply:
This is still here. Please remove